File: Assignment-33/Program1.java
import java.util.*;

class Program1
{
      public static void main(String arg[])
      {
            Scanner sobj = new Scanner(System.in);

            System.out.println("Enter The Number:");
            int iNo = sobj.nextInt();

            Digit obj = new Digit();

            int iRet = obj.CountEven(iNo);
            System.out.println("Number of Even Digit are :"+iRet);
      }
}
class Digit
{
      public int CountEven(int iNo)
      {
            int iCnt = 0;
            int iDigit = 0;

            if(iNo < 0)
            {
                  iNo = -iNo;
            }
            while(iNo !=0)
            {
                  iDigit = iNo % 10;
                  
                  if((iDigit % 2)==0)
                  {
                        iCnt++;
                  }

                  iNo = iNo / 10;
            }
            return iCnt;

      }
}

File: Assignment-13/program1.c
#include<stdio.h>

int CountEven(int iNo)
{
      int iDigit = 0;
      int EvenDigit = 0;

      while(iNo!=0)
      {
            iDigit = iNo % 10;

            if((iDigit % 2)==0)
            {
                  EvenDigit++;
            }

            iNo = iNo / 10;
      }
      return EvenDigit;
}
int main()
{
      int ivalue = 0;
      int iRet = 0;

      printf("Enter The Number :");
      scanf("%d",&ivalue);

      iRet = CountEven(ivalue);

      printf("%d",iRet);

      return 0;
}